src/seq_mv/csr_matop_device.c

```diff
@@ -1530,9 +1558,6 @@ hypre_CSRMatrixTransposeDevice(hypre_CSRMatrix  *A,
 #elif defined(HYPRE_USING_ROCSPARSE)
       hypreDevice_CSRSpTransRocsparse(nrows_A, ncols_A, nnz_A, A_i, A_j, A_data, &C_i, &C_j, &C_data,
                                       data);
-#elif defined(HYPRE_USING_ONEMKLSPARSE)
```

Comment:
Just want to make sure if it is supposed to be deleted, since I saw adding SYCL support for transpose on device in `csr_matop.c`.

Reply:
OK. maybe I understand now: this removed oneMKL version of the transpose

Reply:
Correct, since oneMKL doesn't have a sparse transpose function (as of now).